src/domain/state.ts

```diff
@@ -642,15 +649,25 @@ function resetSignedDocument(current: Application) {
   current.sections.signature.signedDocName = '';
 }
 
-function transitionToRejected(current: Application, updatePart: Partial<UpdateApplication>) {
+function transitionToRejected(current: Application, updatePart: Partial<UpdateApplication>, rejectedBy: string) {
   current.state = 'REJECTED';
   current.denialReason = updatePart.denialReason || '';
+  current.updates.push({
+    date: new Date(),
+    type: 'REJECTED',
+    info: { rejectedBy }
```

Comment:
not sure if i'm reading this right - wouldn't it evaluate to `info: { userId: userId }` ?

Reply:
`info: { rejectedBy: rejectedBy }`

Reply:
rejectedBy will be the user Id as passed from the service

Reply:
i'm just confused cuz it looks like this would make the object key the same as the value? 😛 i.e. `info: {"someUserId": "someUserId"}` instead of `info: {rejectedBy: "someUserId"}`

Reply:
lol that's right but weird I didn't notice it when I tested this on postman, let me check it and will fix

Reply:
haha phew...i thought i was missing some sort of js magic here! 😂

Reply:
verified that it works as intended { rejectedBy: '....' }